zebra/zebra_nhg.c

```diff
@@ -3080,6 +3090,11 @@ void zebra_nhg_install_kernel(struct nhg_hash_entry *nhe)
 	/* Resolve it first */
 	nhe = zebra_nhg_resolve(nhe);
 
+	if (zebra_nhg_set_valid_if_active(nhe)) {
+		if (IS_ZEBRA_DEBUG_NHG_DETAIL)
+			zlog_debug("Setting the valid flag for nh %pNG", nhe);
```

Comment:
The question is just for consistency: zebra_nhg_handle_install has `__func__`, here - not. Which one to follow?

Reply:
ack.